### ISSUE TITLE
Issue with typed constants

### ISSUE THREAD

**Issue by TeacanTzu** [godotengine/godot#76695]
### Godot version

4.0.beta1

### System information

Windows 11

### Issue description

Passing a constant declared like the following
const f: float = 1
will be considered as an integer inside tween.tween_property()

these work:
const f: float = 1.0
var f: float = 1

### Steps to reproduce

Creating for example a sprite in a 2d project and adding this scripts highlights the error:

```
extends Node

const FLOAT : float = 1
const FLOAT2 : float = 1.0
var FLOAT3 : float = 1

var i : int = 2

func _ready():
    print(FLOAT / i)     # expected result of 0.5 but instead received result of 0
    print(FLOAT2 / i)     # received the expected result of 0.5
    print(FLOAT3 / i)     # received the expected result of 0.5
```

**Comment by KoBeWi** [comment]
I checked the project and I don't see any problem. All 3 methods work the same.

> 4.0.beta1

If you are really using that version, upgrade to stable (4.0.2), because it might be a bug that was fixed long ago.

**Comment by TeacanTzu** [comment]
fixed in (4.0.2)

**Comment by dalexeev** [comment]
Duplicate of #70861. Fixed by #71844.
